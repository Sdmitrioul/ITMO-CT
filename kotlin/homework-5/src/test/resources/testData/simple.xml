<mediawiki xml:lang="ru">
    <page>
        <title>Простой заголовок with some English words</title>
        <revision>
            <timestamp>2020-01-01T00:00:00Z</timestamp>
            <text bytes="65">[[Простой как-бы текст]] ну и English words!</text>
        </revision>
    </page>
</mediawiki>