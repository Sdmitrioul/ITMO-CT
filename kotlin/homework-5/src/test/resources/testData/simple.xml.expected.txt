Топ-300 слов в заголовках статей:
1 заголовок
1 простой

Топ-300 слов в статьях:
1 как
1 простой
1 текст

Распределение статей по размеру:
1 1

Распределение статей по времени:
2020 1
